admin_manual/ai/app_assistant.rst

```diff
@@ -63,6 +63,20 @@ In order to make use of text processing features in the assistant, you will need
 * :ref:`llm2<ai-app-llm2>` - Runs open source AI language models locally on your own server hardware (Customer support available upon request)
 * *integration_openai* - Integrates with the OpenAI API to provide AI functionality from OpenAI servers  (Customer support available upon request; see :ref:`AI as a Service<ai-ai_as_a_service>`)
 
+These apps currently implement the following Assistant Tasks:
+
+* *Generate text* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Summarize* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Generate headline* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Extract topics* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+
+Additionally, *integration_openai* also implements the following Assistant Tasks:
+
+* *Context write* (Tested with OpenAI GPT-3.5)
+* *Reformulate text* (Tested with OpenAI GPT-3.5)
```

Comment:
It implements more task types:
* Translate
* Generate image
* Audio to text

Reply:
But this is the Text processing section